### PR TITLE
Feature/brainy

### PR DESCRIPTION
**Pull request by jlconlin**
Here we are doing the opposite of "dimwits"—being "brainy". I'm removing the units and thus dimwits from the code/repository.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -4,12 +4,15 @@
...
 	url = https://github.com/njoy/variant-adapter.git
 [submodule "dependencies/ENDFtk"]
 	path = dependencies/ENDFtk
 	url = https://github.com/njoy/ENDFtk.git
+[submodule "dependencies/json-adapter"]
+	path = dependencies/json-adapter
+	url = https://github.com/njoy/json-adapter.git
+[submodule "dependencies/hana-adapter"]
+	path = dependencies/hana-adapter
+	url = https://github.com/njoy/hana-adapter.git
```
Comment (whaeck) [inline .gitmodules]: Why was hana added as a dependency?

Comment (jlconlin) [reply .gitmodules]: Yeah, I can’t remember why I added that. I’ll remove it.

**Comment by jlconlin** [comment]
There are a few reasons to remove dimwits:
- Like with several of our other libraries (ENDFtk), we are simply returning what is given on the file. There are implied units, but nothing explicit.
- It greatly simplifies exposing this to external modules. Keeping the units makes it much harder to serialize to JSON allowing other modules to get the input parameters. Also exposing these things to Python would be challenging if we still had dimwits.

File: .gitmodules (hunk 1 above)
Comment (jlconlin) [reply .gitmodules]: Hana is used in PLOTR. We used to get it implicitly from dimwits, but now that is removed I had to add it.

Now, maybe we could get rid of hana, but we need it for now.

**Pull request merged**